pom.xml

```diff
@@ -242,6 +242,15 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
This is because of GlobalTracer, Am I right?

Reply:
If https://github.com/opentracing-contrib/java-jdbc/blob/master/src/main/java/io/opentracing/contrib/jdbc/TracingConnection.java could be configured with tracer we wouldn't have this problem.

Reply:
@malafeev could you please leave a comment why it is there?

Reply:
@pavolloffay No, it is not related to GlobalTracer.
I think the problem that spring tests shares context.
That's why standalone test passes but `mvn test` failes (#61)

Reply:
<code>
spanIsCreatedForPreparedStatement(io.opentracing.contrib.spring.cloud.jdbc.JdbcTracingTest)  Time elapsed: 0.002 sec  <<< FAILURE!
java.lang.AssertionError: expected:<1> but was:<0>
</code>

Reply:
GlobalTracer is used in jdbc tracing because of:
```
static {
    try {
      DriverManager.registerDriver(INSTANCE);
    } catch (SQLException e) {
      throw new IllegalStateException("Could not register TracingDriver with DriverManager", e);
    }
  }
```

I need static instance of driver.